docs/source/usersguide/input.rst

```diff
@@ -408,7 +402,16 @@ The ``<run_cmfd>`` element indicates whether or not CMFD acceleration should be
 turned on or off. This element has no attributes or sub-elements and can be set
 to either "false" or "true".
 
-  *Defualt*: false
+  *Default*: false
+
+``<run_mode>`` Element
+----------------------
+
+The ``<run_mode>`` element indicates which run mode should be used when OpenMC
+is executed. This element has no attributes or sub-elements and can be set to
+"eigenvalue", "fixed source", "plot", "volume", or "particle restart".
+
+  *Default*: None
```

Comment:
Maybe, maybe not. We currently do not have a default. The MCNP default is effectively fixed source, so eigenvalue might come as a surprise to some users.